recipes/chimes-calculator/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "chimes-calculator" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
```

Comment:
No need to set a variable that will be used in a single place. Just put the number there in the bn field.